src/codegen/utilities/helpers.py

```diff
@@ -10,6 +10,35 @@
     re.compile("([^_0-9])([0-9])"),
 ]
 
+PascalTokenSubstitution = namedtuple(
+    "PascalTokenSubstitution", ["pascal_representation", "preferred_representation"]
+)
+
+SPECIAL_CASE_PASCAL_TOKENS = [
+    # NI uses UInt, not Uint, and never U_INT when converting to snake.
+    PascalTokenSubstitution("Uint", "UInt"),
+]
```

Comment:
FYI, a `dict` would be fine too. 

I see you copied this from https://github.com/ni/grpc-device/blob/main/source/codegen/common_helpers.py